packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -79,19 +81,31 @@ export default function ClassifierContainer({
   const allowedWorkflows = canPreviewWorkflows ? project?.links.workflows : project?.links.active_workflows
   const allowedWorkflowID = allowedWorkflows.includes(workflowID) ? workflowID : null
 
+  /** Fetch the workflow object by id using SWR */
   const workflowSnapshot = useWorkflowSnapshot(allowedWorkflowID)
+
+  /** Fetch workflow task strings using SWR. Locale is passed from app-project */
```

Comment:
```suggestion
  /*
  Fetch workflow strings using SWR. Locale is passed from component props.
 */
```

Reply:
Small grammar thing: `/** */` is used for docgen comments. [JS block comments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#block_comments) use `/* */`.